README.md

```diff
@@ -122,13 +122,23 @@ Compiling requires using CMake 3.1 or newer. GCC and Clang are both known to wor
 
 #### Docker building
 
-The recommended way to build for most platforms is to use Docker. Several Docker images are provided that contain the requisite toolchain and dependencies for building mGBA across several platforms.
+The recommended way to build for most platforms is to use Docker. Several Docker images are provided that contain the requisite toolchain and dependencies for building mGBA across several platforms. 
+
+Note: If you are on Windows, you need to make sure your terminal session is inside the mGBA repository root directory before using Docker. (See issue [#1985](https://mgba.io/i/1985) for details.) If you are on an older Windows system, you may need to configure your Docker to use VirtualBox shared folders to correctly map your current `mgba` checkout directory to the Docker image's working directory.
+
+For example, on Windows, after cloning the repository, we will assume the `mgba` checkout directory is located at `X:\github\mgba`. What you want to do is to open up the terminal (Powershell or WSL) and navigate to the `mgba` root directory before proceeding:
```

Comment:
I feel like this is still a misunderstanding of what's going on. I think what you're trying to explain here is just to change directory into the directory the checkout is in. This applies to whatever OS or shell you're on. That's just to get the `$PWD` working later on. It doesn't need to be `$PWD`, that was just to simplify the steps. But I feel like this is de-simplifying it now.

Reply:
Removing all those explanations. I supposed I didn't really understand what my dev machine's problem was, and why I sought out that Github issue in the first place to get my mGBA build up and working.